web/src/lib/components/StallView.svelte

```diff
@@ -298,31 +298,31 @@
                     {showItemsOwner} {showItemsCampaign}
                     card={ItemCard}
                     style={ListViewStyle.List}>
-                    <div slot="new-entity" class="lg:flex justify-center lg:w-2/3 mx-auto" let:setCurrent={setCurrent}>
+                    <div slot="new-entity" class="lg:flex items-center lg:justify-start justify-center" let:setCurrent={setCurrent}>
                         {#if isCampaignStall}
-                            <div id="auction-hour-1" class="grid place-items-center w-full mx-auto my-10 p-4" on:click|preventDefault={() => loginAndNewItem(setCurrent, () => new TimeAuction())}>
-                              <div class="w-full my-8 grid place-items-center">
-                                <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width={1.5} stroke="currentColor" class="w-24 h-24">
+                            <div id="auction-hour-1" class="" on:click|preventDefault={() => loginAndNewItem(setCurrent, () => new TimeAuction())}>
+                              <div class="grid place-items-center">
+                                <!-- <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width={1.5} stroke="currentColor" class="w-24 h-24">
                                   <path stroke-linecap="round" stroke-linejoin="round" d="M12 6v6h4.5m4.5 0a9 9 0 11-18 0 9 9 0 0118 0z" />
-                                </svg>
-                                <p class="btn btn-primary font-bold text-center">1 Hour of your time</p>
+                                </svg> -->
+                                <button class="btn btn-primary font-bold text-center w-52">1 Hour of your time</button>
                               </div>
                             </div>
                         {/if}
 
-                        <div id="anchorIdAuctionItem" class="grid place-items-center w-full mx-auto my-10 p-4" on:click|preventDefault={() => loginAndNewItem(setCurrent, () => new Auction())}>
-                          <div class="grid place-items-center w-full my-8">
-                            <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-24 h-24">
+                        <div id="anchorIdAuctionItem" class="grid lg:place-items-start place-items-center w-full p-4" on:click|preventDefault={() => loginAndNewItem(setCurrent, () => new Auction())}>
+                          <div class="w-full flex lg:justify-start justify-center">
+                            <!-- <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-24 h-24">
```

Comment:
Are we removing the icons now? When was this decided? I might have missed it, but these icons were good UX I think!